examples/sklearn_penguins/requirements.txt

```diff
@@ -1,3 +1,3 @@
 scikit-learn==0.23.2
-tensorflow==2.5.1
+tensorflow==2.4.1
```

Comment:
@casassg Why the downgrade of TF?

Reply:
TFX 0.30 is compatible only w tf 2.4. If upgrade then requirements.txt doesnt resolve

Reply:
Why are we still using TFX 0.30?

Reply:
not sure if example will work out of the box on tfx 1.0.0 and its out scope for this PR to update the examples themselves

Reply:
The example should be compatible with TFX 1.0.0, it just wasn't released yet when the example was merged in. Would prefer upgrading TFX over downgrading TF. Let me know if it doesn't work out of the box.

Reply:
there we go! upgraded. Had to fix an issue that a test was picking up the tfx version of the example instead of the local version, but all good now!